recipes/fastpca/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "FastPCA" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/moldyn/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 67aebbbfb637f8a0fdb1ae130e3f457b3a21fc982cb99de7e61b4fe03dcbd0ae
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - boost
+
+  host:
+    - libopenblas
```

Comment:
can you follow https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas

Reply:
yes, thanks